assignments/prototypes.js

```diff
@@ -32,7 +47,20 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
- 
+function Humanoid (attributes) {
+  CharacterStats.call(this,attributes);
+  this.team = attributes.team;
+  this.weapons = attributes.weapons;
+  this.language = attributes.language;
+  this.name = attributes.name;
+  this.dimensions = attributes.dimensions;
+  this.createdAt = attributes.createdAt;
+  this.healthPoints = attributes.healthPoints;
```

Comment:
Instead of inheriting from another constructor you are declaring all properties here. 
 
`this.team = attributes.team;`
`this.weapons = attributes.weapons;`
`this.language = attributes.language;`
Only these needs to survive here!